www/md_base/guanlecoja/config.coffee

```diff
@@ -3,8 +3,8 @@
 #   This module contains all configuration for the build process
 #
 ### ###############################################################################################
-ANGULAR_TAG = "~1.3.15"
-ANGULAR_MATERIAL_TAG = "~0.8.3"
+ANGULAR_TAG = "~1.4.0"
+ANGULAR_MATERIAL_TAG = "~0.10.0"
```

Comment:
Aha, you have changed the tag to `0.10.0`. I haven't migrate `md_base` to it yet. Hope there won't be too much problem to resolve.